Gemfile.lock

```diff
@@ -65,20 +65,21 @@ GEM
     diff-lcs (1.1.3)
     equalizer (0.0.11)
     erubis (2.7.0)
-    ethon (0.9.1)
+    ethon (0.10.1)
```

Comment:
These got bumped by the Ruby system, I am compiling with:
```bundle install; jbundle install
bundle exec rake killbill:clean;bundle exec rake build;bundle exec rake killbill:package```

Is there a better way that doesn't result in the versions changing?